config/config.php

```diff
@@ -1,14 +1,5 @@
 <?php
 
-/*
- * This file is part of jwt-auth.
- *
- * (c) Sean Tymon <tymon148@gmail.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
```

Comment:
Removed in purpose here? Just double checking

Reply:
Yes, cause this file will end up in the end-user repositories after running `vendor:publish`

Reply:
Makes very much sense to me 👍🏼